src/pcapi/routes/serialization/stock_serialize.py

```diff
@@ -9,7 +11,8 @@
 from pcapi.serialization.utils import to_camel
 
 
-class StockCreationBodyModel(BaseModel):
+# DEPRECATED: (venaud, 2021-01-19): to remove when new offer and stock creation is deployed
+class StockCreationBodyModelOld(BaseModel):
```

Comment:
```suggestion
class StockCreationBodyModelDeprecated(BaseModel):
```
Pourquoi ne pas mettre directement deprecated dans le nom ?